vsts/node-nightly-df.yaml

```diff
@@ -92,8 +98,12 @@ phases:
       IOT_PROVISIONING_ROOT_CERT: $(WINDOWS-IOT-PROVISIONING-ROOT-CERT)
       IOT_PROVISIONING_ROOT_CERT_KEY: $(WINDOWS-IOT-PROVISIONING-ROOT-CERT-KEY)
       IOT_PROVISIONING_SERVICE_CONNECTION_STRING: $(WINDOWS-IOT-PROVISIONING-SERVICE-CONNECTION-STRING)
+      DPS_CONN_STRING_INVALID_CERT: $(DPS-CONN-STRING-INVALID-CERT)
+      DPS_GLOBAL_DEVICE_ENDPOINT_INVALID_CERT: $(DPS-GLOBAL-DEVICE-ENDPOINT-INVALID-CERT)
+      IOTHUB_CONN_STRING_INVALID_CERT: $(IOTHUB-CONN-STRING-INVALID-CERT)
+      IOTHUB_DEVICE_CONN_STRING_INVALID_CERT: $(IOTHUB-DEVICE-CONN-STRING-INVALID-CERT)
 
-  - task: PublishTestResults@1
+  - task: PublishTestResults@2
```

Comment:
>task: PublishTestResults@2 [](start = 4, length = 26)

not understanding what requires this change

Reply:
Because it started failing with a path or filename too long.  Searching the web, found that vsts knew about this and said to go to the newer version.

---
In reply to: [221354330](https://github.com/Azure/azure-iot-sdk-node/pull/378#discussion_r221354330) [](ancestors = 221354330)